daemon/cmd/datapath.go

```diff
@@ -209,51 +219,58 @@ func (d *Daemon) syncHostIPs() error {
 			}
 
 			if len(ip) > 0 {
-				specialIdentities = append(specialIdentities,
+				specialIdentities = append(specialIdentities, ipIDLabel{
 					identity.IPIdentityPair{
 						IP: ip,
 						ID: identity.ReservedIdentityHost,
-					})
+					},
+					labels.LabelHost,
+				})
 			}
 		}
 
-		specialIdentities = append(specialIdentities,
+		specialIdentities = append(specialIdentities, ipIDLabel{
 			identity.IPIdentityPair{
 				IP:   net.IPv6zero,
 				Mask: net.CIDRMask(0, net.IPv6len*8),
 				ID:   identity.ReservedIdentityWorld,
-			})
+			},
+			labels.LabelWorld,
+		})
 	}
 
 	existingEndpoints, err := lxcmap.DumpToMap()
 	if err != nil {
 		return err
 	}
 
-	for _, ipIDPair := range specialIdentities {
-		isHost := ipIDPair.ID == identity.ReservedIdentityHost
+	daemonResourceID := ipcachetypes.NewResourceID(ipcachetypes.ResourceKindDaemon, "", "")
+	for _, ipIDLblsPair := range specialIdentities {
+		isHost := ipIDLblsPair.ID == identity.ReservedIdentityHost
 		if isHost {
-			added, err := lxcmap.SyncHostEntry(ipIDPair.IP)
+			added, err := lxcmap.SyncHostEntry(ipIDLblsPair.IP)
 			if err != nil {
 				return fmt.Errorf("Unable to add host entry to endpoint map: %s", err)
 			}
 			if added {
-				log.WithField(logfields.IPAddr, ipIDPair.IP).Debugf("Added local ip to endpoint map")
+				log.WithField(logfields.IPAddr, ipIDLblsPair.IP).Debugf("Added local ip to endpoint map")
 			}
 		}
 
-		delete(existingEndpoints, ipIDPair.IP.String())
+		delete(existingEndpoints, ipIDLblsPair.IP.String())
 
+		lbls := ipIDLblsPair.Labels
 		// Upsert will not propagate (reserved:foo->ID) mappings across the cluster,
 		// and we specifically don't want to do so.
```

Comment:
Is the comment still valid? It refers to `Upsert()`, but we're not using it anymore.

Reply:
Good point, I think the comment was kinda stale even before this change. I've removed it.